updated the image in Releasemanagement

werner-roman opened the pull request:
Update image in release management to indicate that the version names after Jupiter are examples

changes the link to the image to a relative link (against BP) -> otherwise, it doesn't render in the MD Pre-view.

stephanbcbauer commented on .markdownlint.json:
```diff
@@ -1,13 +1,19 @@
     ]
-  }
+  },
+  "MD034": false,
```
deactivating these requirements is needed?

jSchuetz88 replied:
For MD034, MD036 and MD040: These are technically optional and are not needed, but while MD034 and MD 040 are fixable in short notice, I couldn't provide a fix for MD034 for hundreds of bare URLs (at least not until the end of the next week).

jSchuetz88 replied:
For MD028: Unfortunately: In a lot of standards intentionally used for multiple individual quotes. But this one would also be fixable without effecting the contents.

stephanbcbauer commented on .markdownlint.json:
```diff
@@ -1,13 +1,19 @@
   },
+  "MD028": false,
   "MD033": {
     "allowed_elements": [
```
To be honest, i don't like to allow these elements

jSchuetz88 replied:
I removed all HTML-Tags where it was possible to do so without effecting the content or actual structure. But as MD-Files dont allow multi-lines or bullet points within tables and I can not change the structure that is connected to normative CACs retroactively. 

We can only prevent the usage of these html-tags for all future releases, but we can not change them retroactively.

stephanbcbauer reviewed (commented):
Overfall LGTM, what we should discuss is the naming of the pictures. I would prefer something without blanks and small letters. Instead of CX 0002_Picture1.jpg maybe a more meaningful name. Like cx-0002_what-is-the-image-about.jpg

jSchuetz88 commented:
> Overfall LGTM, what we should discuss is the naming of the pictures. I would prefer something without blanks and small letters. Instead of CX 0002_Picture1.jpg maybe a more meaningful name. Like cx-0002_what-is-the-image-about.jpg

Sorry I missed that one, most of the time I changed the naming to the kebab-case convention. I will update those.

Commit:
# updated the image in Releasemanagement
# changes the link to image to relative link -> otherwise it doesnt render in the MD Pre-view.

stephanbcbauer commented on src/pages/release-management.md:
```diff
@@ -4,7 +4,7 @@ Effective management of versions and changes for Catena-X standards, KITs and op
 ## Support Dates
 
-![Support Dates](/img/support-dates.jpg)
+![Support Dates](./../../static/img/support-dates.jpg)
```
```suggestion
![Support Dates](/img/support-dates.jpg)
```

stephanbcbauer commented on src/pages/release-management.md:
```diff
@@ -38,7 +38,7 @@ Based on the first experiences in live operation, the release management for the
 ## Rationale for Changes
 
-![Release Timeline](/img/release-timeline.jpg)
+![Release Timeline](./../../static/img/release-schedule.png)
```
```suggestion
![Release Timeline](/img/release-schedule.png)
```

stephanbcbauer reviewed (approved):
LGTM

stephanbcbauer committed:
Update src/pages/release-management.md

stephanbcbauer committed:
Update src/pages/release-management.md
